inhouse/command_handlers/soloqueue_leaderboard.py

```diff
@@ -34,101 +55,242 @@ def add_player(self,name,tier,rank,lp):
+                    rank_string = ""
+                    name_string = ""
+                    num_string = ""
         for rank in self.iron_dict:
             for player in sorted(self.iron_dict[rank], key = itemgetter(1), reverse=True):
-                total_string += f"{player[0]}: Iron {rank} {str(player[1])} LP\n"                
+                name_string += f"{ironEmoji} {player[0]}\n"
+                rank_string += f"Iron {rank} {str(player[1])} LP\n"                
+                num_string += f"{self.num_ranked_past_week(player[0])}\n"
                 i += 1
-                if i == 10:
-                    i = 0
-                    msg = discord.Embed(description=total_string, color=discord.Color.gold())
+                if i % 10 == 0:
+                    msg = discord.Embed(color=discord.Color.blue())
+                    if i == 10:
+                        msg.title = ":trophy: **Soloqueue Standings** :trophy:"
+                    msg.add_field(name="Summoner", value=name_string, inline=True)
+                    msg.add_field(name="Rank", value=rank_string, inline=True)
+                    msg.add_field(name="Games in Past Week", value=num_string, inline=True)
                     msg_list.append(msg)
-                    total_string = ""
+                    rank_string = ""
+                    name_string = ""
+                    num_string = ""
         for player in self.unranked_dict:
-            total_string += f"{player[0]}: Unranked"
+            name_string += f"{unratedEmoji} {player[0]}\n"
+            rank_string += f"Unranked\n"
+            num_string += f"{self.num_ranked_past_week(player[0])}\n"
             i += 1
-            if i == 10:
-                i = 0
-                msg = discord.Embed(description=total_string, color=discord.Color.gold())
+            if i % 10 == 0:
+                msg = discord.Embed(color=discord.Color.blue())
+                if i == 10:
+                    msg.title = ":trophy: **Soloqueue Standings** :trophy:"
+                msg.add_field(name="Summoner", value=name_string, inline=True)
+                msg.add_field(name="Rank", value=rank_string, inline=True)
+                msg.add_field(name="Games in Past Week", value=num_string, inline=True)
                 msg_list.append(msg)
-                total_string = ""
+                rank_string = ""
+                name_string = ""
+                num_string = ""
         if i != 0:
-            msg = discord.Embed(description=total_string, color=discord.Color.gold())
+            msg = discord.Embed(color=discord.Color.blue())
+            msg.add_field(name="Summoner", value=name_string, inline=True)
+            msg.add_field(name="Rank", value=rank_string, inline=True)
+            msg.add_field(name="Games in Past Week", value=num_string, inline=True)
             msg_list.append(msg)
-        return msg_list
\ No newline at end of file
+        return msg_list
+    @tasks.loop(hours=3)
```

Comment:
nit: can we move this to a constant somewhere for easier config from a file in the future?